schemas/json/block.json

```diff
@@ -377,6 +377,21 @@
 									}
 								}
 							]
+						},
+						"blockGap": {
+							"default": false,
+							"oneOf": [
+								{
+									"type": "boolean"
+								},
+								{
+									"type": "array",
+									"items": {
+										"type": "string",
+										"enum": [ "vertical", "horizontal" ]
```

Comment:
I know you've already opened up a discussion about it in #47518 and #47523, but currently this array only supports including both of these values, it doesn't support only including one direction.